mythril/analysis/report.py

```diff
@@ -64,6 +63,24 @@ def __init__(
         self.source_mapping = None
         self.discovery_time = time() - StartTime().global_start_time
         self.bytecode_hash = get_code_hash(bytecode)
+        if transaction_sequence is None:
+            self.tx_sequence_users = None
```

Comment:
why are these variables?

Reply:
tx_sequence_users is used for displaying for users with "<blockTime>" sort of field than some concrete field. (tx_sequence_jsonv2 is for maestro)

Reply:
But what's the reason for storing them as class variables versus just calculating when they are displayed

Reply:
the function self.add_block_data, and also json.dumps() should be written in multiple places if that's the case, in this case it's just easily reusing them.

Reply:
I can find only one use of the variables. 
My main point is that the actual data is in transaction_sequence. 
Creating two variables which maintain some representation of the same data doesn't seem like a good approach

I'd either store the initial data, and do the conversion when you need it in some function. 
Or create some functions which do something like
```python
@property
def tx_sequence_users(self):
    return json.dumps(self.transaction_sequence, indent=4) if self.transaction_sequence else None
```